src/components/custom/_eu-redirect-modal.tsx

```diff
@@ -8,6 +7,17 @@ import device from 'themes/device'
 // Icons
 import Close from 'images/svg/custom/close-2.svg'
 
+type EURedirectProps = {
+    aria_label?: string
+    closeModal?: () => void
+    is_open: boolean
+    ref?: Ref<HTMLAnchorElement>
+    rel?: string
+    target?: string
+    to?: string
+    toggle: () => void
```

Comment:
```suggestion
    toggle: React.MouseEventHandler<HTMLDivElement> | React.MouseEventHandler<HTMLImageElement>
```
what do you thing about this approach? 
or 
```suggestion
    toggle: React.MouseEventHandler<HTMLElement>
```
or
```suggestion
    toggle: (event:  React.MouseEvent<HTMLElement>) => void 
```

Reply:
Cool idea @michio-rework 👍 
Let my try to implement it

Reply:
Please check again 🙏